src/plugins/intel_cpu/src/nodes/roi_align.cpp

```diff
@@ -895,13 +932,17 @@ void ROIAlign::executeSpecified() {
             IE_THROW() << "Demanded batch (id = " << roiBatchInd << ") doesn't exist";
         }
 
-        float x1 = srcRoiPtr[0] * spatialScale;
-        float y1 = srcRoiPtr[1] * spatialScale;
-        float x2 = srcRoiPtr[2] * spatialScale;
-        float y2 = srcRoiPtr[3] * spatialScale;
+        float x1 = (srcRoiPtr[0] + offset_src) * spatialScale + offset_dst;
+        float y1 = (srcRoiPtr[1] + offset_src) * spatialScale + offset_dst;
+        float x2 = (srcRoiPtr[2] + offset_src) * spatialScale + offset_dst;
+        float y2 = (srcRoiPtr[3] + offset_src) * spatialScale + offset_dst;
 
-        float roiHeight = std::max(y2 - y1, 1.0f);
-        float roiWidth = std::max(x2 - x1, 1.0f);
+        float roiHeight = y2 - y1;
+        float roiWidth = x2 - x1;
+        if (!aligned) {
+            roiHeight = std::max(roiHeight, 1.0f);
```

Comment:
The the computing of std::max(roiHeight, 1.0f) depends on the flag "aligned", while in the spec decription this computation is independent. Shouldn't we keep the same behavior, or revise the decription in the spec? https://github.com/liubo-intel/openvino/blob/liiubo/roi_align_9_spec/docs/ops/detection/ROIAlign_9.md#:~:text=roi_width%20%3D%20max(spatial_scale%20*%20(x_2%20%2D%20x_1)%2C%201.0)%2C%20roi_height%20%3D%20max(spatial_scale%20*%20(y_2%20%2D%20y_1)%2C%201.0)

Reply:
Hi, @xuchen-intel : thanks for pointing this out, previously we forgot to modify this in spec, now have updated this description:
https://github.com/openvinotoolkit/openvino/commit/9408ce16d9861a04503822116304d1fb47ec232c